src/lib/support/CHIPMem-Malloc.cpp

```diff
@@ -26,8 +26,13 @@
 #include <core/CHIPConfig.h>
 #include <support/CHIPMem.h>
 
+#include <cstring>
 #include <stdlib.h>
 
+#ifndef WEAK
```

Comment:
same here and below for CHIP-Mem - I assume WEAK is not used anymore after reorganizing the code.

Reply:
Yes, you are right